### DISCUSSION TITLE
AHK++ sees AHK v1 code as v2 code in VS Code

### DISCUSSION THREAD

**Discussion by jeffwirth** [mark-wiemer/ahkpp#408] [Q&A] [answered]
When I install AHK++ in VS Code, it doesn't show anything in the outline area of the Explorer panel. I also get an error message when trying to get help that shows a  path that seems to be for AHK v2 instead of V1. Is there a place that I need to set which version of AHK I'm running? Thanks for any help you can provide.

**Comment by fade2gray** [comment, marked as answer]
See https://github.com/mark-wiemer-org/ahkpp/issues/396#issuecomment-1676525716

**Reply by jeffwirth** [reply]
Thanks! That did the trick. I appreciate your quick reply.

**Comment by mark-wiemer** [comment]
Ref #445 as well :)
